build_tools/ci/run_matmul_test.sh

```diff
@@ -120,12 +144,28 @@ function run_matmul_test() {
       --iree-amd-aie-peano-install-dir=${peano_install_path} \
       --iree-amd-aie-mlir-aie-install-dir=${mlir_aie_install_path} \
       --iree-amd-aie-vitis-install-dir=${vitis_path} \
+      --iree-hal-dump-executable-files-to=$PWD \
+      --iree-hal-dump-executable-intermediates-to=$PWD \
```

Comment:
The `intermediates` flag is redundant here. `files` is a meta-flag that sets all the other flags: https://iree.dev/developers/general/developer-tips/#dumping-executable-files